governance/src/main/java/org/apache/servicecomb/governance/marker/RequestProcessor.java

```diff
@@ -103,4 +115,68 @@ private boolean operatorMatch(String str, RawOperator rawOperator) {
     }
     return true;
   }
+
+  private boolean customMatch(GovernanceRequest request, Matcher matcher) {
+    if (matcher.getCustomMatcher() == null) {
+      return true;
+    }
+    String customMatcherHandlerName = matcher.getCustomMatcher().getCustomMatcherHandler();
+    String customMatcherParameters = matcher.getCustomMatcher().getCustomMatcherParameters();
+
+    if (StringUtils.isEmpty(customMatcherHandlerName) || StringUtils.isEmpty(customMatcherParameters)) {
+      return true;
+    }
+
+    CustomMatch customMatcherHandler = generateHandler(customMatcherHandlerName);
+    return customMatcherHandler.matchRequest(request, customMatcherParameters);
+  }
+
+  private CustomMatch getBeanByHandlerName(String customMatcherHandler) {
+    Object extractObject = null;
+    if (applicationContext.containsBean(customMatcherHandler)) {
+      extractObject = applicationContext.getBean(customMatcherHandler);
+      if (!(extractObject instanceof CustomMatch)) {
+        LOGGER.error("{} {}", customMatcherHandler, CustomMatch.errorMessageForNotImplements);
+        throw new RuntimeException(customMatcherHandler + CustomMatch.errorMessageForNotImplements);
+      }
+      return (CustomMatch)extractObject;
+    }
+    return null;
+  }
+
+
+  public CustomMatch generateHandler(String customMatcherHandler) {
+    CustomMatch extractObject = getBeanByHandlerName(customMatcherHandler);
+    if (extractObject != null) {
+      return extractObject;
+    }
+
```

Comment:
It could be better if we can add a info log to show we create a extractionHandler here.

Reply:
done